portal/src/components/Materials/Navigation/Navigation.component.js

```diff
@@ -30,7 +30,7 @@ export default {
      */
     links() {
       const { materials, material, materials_loading } = this
-      if (materials) {
+      if (materials && materials.search_text) {
```

Comment:
Ok, freaky. Ik wist niet dat materials dat hadden. Wel iets waar we afscheid van willen nemen, zoals eerder besproken (materials store kan de deur uit). Als het niet anders kan, prima voor nu.

Reply:
Haha ja. Het is helemaal een vreemd component, dit was even een quick fix 😉 .